### DISCUSSION TITLE
Map view causing SplitView to change dimensions

### DISCUSSION THREAD

**Discussion by kylepinionseb** [stevengharris/SplitView#43] [General]
I'm evaluating this library and I found a problem when trying to put a Map (SwiftUI) into the Top view on a VSplit. I've reproduced it by adding a Map to the Demo sample `simpleAdjustable` Code below.  Once I move the map (or zoom) the Split is resized back to an original position instead of retaining where the User left it. Is there a known workaround to resolve this? Video below.

```
case .simpleAdjustable:
                Split(
                    primary: {
                        Map(coordinateRegion: $region, annotationItems: annotations) {
                            MapPin(coordinate: $0.coordinate)
                        }
                    },
                    secondary: { Color.red }
                )
                .styling(color: .yellow)
                .layout(demo.holders[0].layout)
                .hide(demo.holders[0].hide)

```
https://github.com/user-attachments/assets/d35b85fd-f8d9-442b-b2ec-c22bd211a7ce

Other supporting code:

```
@State var region: MKCoordinateRegion = MKCoordinateRegion(center: CLLocationCoordinate2D(latitude: 37.09, longitude: -95.71), latitudinalMeters: Measurement(value: 15, unit: UnitLength.miles).converted(to: .meters).value, longitudinalMeters: Measurement(value: 15, unit: UnitLength.miles).converted(to: .meters).value)
let annotations: [City] = [ ]
```
   
```
struct City: Identifiable {
    let id = UUID()
    let name: String
    let coordinate: CLLocationCoordinate2D
}
```

**Comment by stevengharris** [comment]
I will take a look, thanks.

**Comment by stevengharris** [comment]
I moved this discussion to an Issue, https://github.com/stevengharris/SplitView/issues/44,  because it's easier for me to track, and I proposed a quick fix.
